cmake/Modules/opm-core-prereqs.cmake

```diff
@@ -42,5 +42,8 @@ set (opm-core_DEPS
 	"opm-parser REQUIRED"
 	# the code which implements the material laws
 	"opm-material REQUIRED"
+	# the code which implements the output routines
 	"opm-output REQUIRED"
+	# the code which implements grids
+	"opm-grid REQUIRED"
```

Comment:
didn't the PR description say that opm-core does not depend on opm-grid? (it seems like that was it a typo: s/not/now/)

slightly off-topic: does this mean that the opm-core removal finally gets serious?